src/cache_arrays.cpp

```diff
@@ -26,10 +26,12 @@
 #include "cache_arrays.h"
 #include "hash.h"
 #include "repl_policies.h"
+#include <inttypes.h>
```

Comment:
not clear why these are needed. There should be no changes to cache_arrays.cpp in this pull request.